foundations/html_css/flexbox/flexbox-growing-and-shrinking.md

```diff
@@ -56,6 +56,10 @@ An important implication to notice here is that when you specify `flex-grow` or
 
 `flex-basis` simply sets the initial size of a flex item, so any sort of `flex-grow`ing or `flex-shrink`ing starts from that baseline size. The shorthand value defaults to `flex-basis: 0%`. The reason we had to change it to `auto` in the `flex-shrink` example is that with the basis set to `0`, those items would ignore the item's width, and everything would shrink evenly. Using `auto` as a flex-basis tells the item to check for a width declaration (`width: 250px`).
 
+### flex-auto
+
+`flex: auto` is one of the short hands of flex. When `flex: auto` is called it inherits the values of `flex-grow: 1 flex-shrink: 1 flex:basis: auto` or `flex: 1 1 auto` using the flex shorthand. Note that `flex: auto` is not the default value when using the flex short hand despite the name being "auto" which may be slightly confusing at first. The default value for flex is `flex: 0 1 auto` or `flex: initial` which we won't get too in depth about here.
```

Comment:
**praise:** I like the note on `flex: auto` not being the default value for flex short hand despite its name as "auto". Good point!

**suggestion:** I think we should drop the final line on "The default value for flex..." There really is no default value of the shorthand property `flex` itself because you have to specify a keyword value for the property. I think it's better to just drop the line and not try to even get into `flex: initial` since the focus on this section is `flex: auto`.

Reply:
Got it! Will do